modules/sandboxed-containers-preparing-openshift-cluster.adoc

```diff
@@ -2,34 +2,35 @@
 //
 // * sandboxed_containers/deploying_sandboxed_containers.adoc
 
-[id="sandboxed-containers-preparing-openshift-cluster_{context}"]
-= Preparing your cluster for {sandboxed-containers-first}
+[id="sandboxed-containers-prerequisites_{context}"]
+= Prerequisites
 
 Before you install {sandboxed-containers-first}, ensure that your {product-title} cluster meets the following requirements:
 
-* Your cluster must be installed on bare metal infrastructure with {op-system-first} workers. Your cluster must use installer-provisioned infrastructure.
-// To Do: Add link to bare metal instructions: ../../installing/installing_bare_metal/installing-bare-metal.adoc#installing-bare-metal[bare metal]
-// To Do: Add link to installer provisioned infrastructure: ../../installing/installing_bare_metal_ipi/ipi-install-overview.adoc#ipi-install-overview[Installer provisioned infrastructure]
+* Your cluster must be installed on bare metal infrastructure with {op-system-first} workers.
+* Your cluster must use installer-provisioned infrastructure.
```

Comment:
We can remove this restriction as discussed. Something along the lines of:

> Your cluster must be installed on on-premise bare metal infrastructure with Red Hat Enterprise Linux CoreOS (RHCOS) workers. To deploy your cluster, you can use any installation method including user-provisioned, installer-provisioned, or assisted installer.

Reply:
Is this a fundamental change? Would this update need to go through change management?

Reply:
Also, I'm not sure we need to mention that they can use any installation method. Barring a prereq saying otherwise, users will assume all installation methods are fine.

Reply:
Is there bare-metal infra that isn't on-prem?

Reply:
>Also, I’m not sure we need to mention that they can use any installation method. Barring a prereq saying otherwise, users will assume all installation methods are fine.

The installation method is okay as long as we have RHCOS nodes. 

My point is that we should remove "Your cluster must use installer-provisioned infrastructure"

Reply:
> Is there bare-metal infra that isn't on-prem?
yes, we can have bare-metal cloud instances, but we are not supporting that today. So good idea to add "on-prem" if we do not already (nice catch).

Reply:
Removing it now. Do you think removing this bullet constitutes a fundamental change that shareholders should be notified about?

Also, you added "on-premise" before bare metal. Is that needed, or can that remain as is?